src/app/pages/blog/article/use-progress.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
You can review as differences [here](https://github.com/openstax/os-webview/pull/2676/commits/0bb425ebc247bc713c5dd5a7be660f5a7e59ed5a#diff-dedf559ba80e06ffe10a9fe116c938fc25a893e784023f31f7c52f67997cf73e). The big-block changes are just reformatting. Other than that, types.